Fundamentals/Setup/Server-Setup/runtime-modes.md

```diff
@@ -55,6 +55,10 @@ Ensure your models are generated by running Umbraco and navigating to **Settings
 
 Fix any compilation errors you might get after this, e.g. if you accidentally referenced deleted models or properties. Running the application will still show the rendered content and you're now ready to optionally enable `Production` mode on your production environment.
 
+:::note
+Ensure you have the `<CopyRazorGenerateFilesToPublishDirectory>true</CopyRazorGenerateFilesToPublishDirectory>` property in your `csproj` project file, so Razor views are always copied to the publish directory. This is required by the CMS to diplay the contents in the backoffice (as read-only when running in Production runtime mode) and for Deploy to be able to compare schemas (otherwise you'll get schema mismatches).
```

Comment:
Should this also mention the property is already set when using the dotnet new template?

Reply:
This property was already included since 9.0.1, so let's not make this note bigger than it already is 😄